test/test_mps.py

```diff
@@ -2198,6 +2198,19 @@ def helper(dtype):
             self.assertEqual(e_string, "MPS does not support cumsum op with int64 input")
 
 
+    def test_cumsum_minus_one_axis(self):
```

Comment:
Can you test the TestConsistency test for cumsum.

Reply:
Also enable that for all types.

Reply:
TestConsistency passes, and expanded this test with datatypes (which all pass).